docs/source/installing_in_virtualenv.rst

```diff
@@ -0,0 +1,165 @@
+Installing Nitrate on RHEL7 (virtualenv) with Apache and MySQL
```

Comment:
In this case how do you feel about removing "RHEL6" from the `installing_in_rhel.rst` file ? It's true the deployment steps can be used for other distros but the reality is that all steps and file locations are specific to one particular distribution, used to create the document.